libraries/AP_GPS/AP_GPS_NMEA.h

```diff
@@ -139,6 +139,8 @@ class AP_GPS_NMEA : public AP_GPS_Backend
     int32_t _new_course;                                        ///< course parsed from a term
     uint16_t _new_hdop;                                                 ///< HDOP parsed from a term
     uint8_t _new_satellite_count;                       ///< satellite count parsed from a term
+    uint8_t _new_quality_indicator;                                     ///< GPS quality indicator parsed from a term
+    AP_GPS::GPS_Status _gga_gps_status = AP_GPS::NO_FIX;
```

Comment:
This needs to be init'd to NO_GPS, or not specified as it will be set to 0 by default in the constructor. As written it will always report a GGA based fix, rather then being able to fall back if GGA is never received.